cmd/destroy/destroy_test.go

```diff
@@ -567,6 +571,238 @@ func TestDestroyWithoutError(t *testing.T) {
 				executor:          executor,
 				nsDestroyer:       destroyer,
 				k8sClientProvider: test.NewFakeK8sProvider(),
+				runner:            &oktetoDefaultRunner{},
+			}
+
+			err := cmd.runDestroy(ctx, opts)
+
+			assert.NoError(t, err)
+			assert.ElementsMatch(t, tt.want, executor.executed)
+			assert.True(t, destroyer.destroyed)
+			assert.True(t, destroyer.destroyedVolumes)
+
+			// check if configmap has been created
+			fakeClient, _, err := cmd.k8sClientProvider.Provide(api.NewConfig())
+			if err != nil {
+				t.Fatal("could not create fake k8s client")
+			}
+			cfg, _ := configmaps.Get(ctx, pipeline.TranslatePipelineName(opts.Name), okteto.Context().Namespace, fakeClient)
+			assert.Nil(t, cfg)
```

Comment:
I think this assert would be more accurate if we check that the error is a not found. cfg could be nil for other reasons, I think any error getting the configmap would return a nil `cfg`

Reply:
Updated. Writing comment since the outdated label has not been added to the comment